docs/faq/edit-a-hip.mdx

```diff
@@ -0,0 +1,125 @@
+---
+id: edit-a-hip
+title: Propose Changes to Existing HIP in GitHub
+pagination_label: Edit an Existing HIP
+sidebar_label: Edit an Existing HIP
+description: Helium Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /edit-a-hip
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+While HIPs are in discussion, the actual text is included in the
+[Helium HIP Repository on GitHub](https://github.com/helium/hip).
+
+Before a HIP goes to a vote, it is likely that changes and revisions will need to be made to an
+existing proposal.
+
+:::tip Key Take Aways
+
+This page will walk users through the process of:
+
+- Creating a personal copy, or "fork", of the Helium HIP repository.
+- Making edits using the in-browser text editor.
+- Submitting these changes for review.
+
+:::
+
+---
+
+## Fork the Repo
+
+:::info What the Fork
+
+Rather than making edits to the main repository, create a personal copy by "forking" the repository
+to your GitHub account.
+
+:::
+
+1. Log in to an existing account or create a new account on [GitHub](https://github.com).
+2. Open the Helium HIP Repository at [https://github.com/helium/hip](https://github.com/helium/hip)
+3. Click the `Fork` button in the upper right-hand corner.
+4. Leave all settings default and confirm using the green `Create Fork` button at the bottom
+   left-hand corner.
+   - A copy of the Helium HIP repository will be made in the user's GitHub account and should open
+     automatically.
+
+<figure className="screensnippet-wrapper">
+  <img
+    src={useBaseUrl('/img/hip/create-fork.gif')}
+    style={{ maxHeight: 400, borderRadius: '40px', padding: '14px 16px 20px' }}
```

Comment:
Oh, I created some styles for this, but locked up in the Sphere docs.
```js
className="add-shadow add-shadow-margin add-border-radius"
```

Reply:
nice! adding an issue for this to track and com eback once sphere docs are merged #1339